django/utils/translation/trans_real.py

```diff
@@ -116,8 +116,19 @@ def _new_gnu_trans(self, localedir, use_null_fallback=True):
         Using param `use_null_fallback` to avoid confusion with any other
         references to 'fallback'.
         """
+        if isinstance(localedir, (tuple, list)):
+            try:
+                localedir, domain = localedir
+            except ValueError:
```

Comment:
Query: Did the system checks added not already catch this error? (i.e. do we need to handle it again here?)